spring-cloud-kubernetes-client-config/src/test/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientBootstrapConfigurationTests.java

```diff
@@ -158,7 +157,7 @@ public void bothPresent() {
 	// tests that @ConditionalOnCloudPlatform(CloudPlatform.KUBERNETES) has the desired
 	// effect, meaning when it is disabled, no property source bean is present
 	@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = Application.class,
-			properties = { "kubernetes.informer.enabled=false" })
+			properties = { "kubernetes.manifests.enabled=false" })
```

Comment:
I will only pin point the important (somehow) changes in this PR. This test was failing, so had to change this setting